sdk/utils/strenum.py

```diff
@@ -1,11 +1,13 @@
 from enum import Enum
+from typing import List
 
 
 class StrEnum(str, Enum):
-    def __str__(self):
+    def __str__(self) -> str:
         """Used when dumping enum fields in a schema."""
-        return self.value
+        ret: str = self.value
+        return ret
 
     @classmethod
-    def list(cls):
-        return list(map(lambda c: c.value, cls))
+    def list(cls) -> List[str]:
+        return list(map(lambda c: c.value, cls))  # type: ignore
```

Comment:
Out of curiosity, does adding ` # type: ignore` have a concrete effect on mypy type checking ?

Reply:
it does. means that mypy doesn't fail on this this line, or even try to analyse it.

to make mypy work on something like this it looks like you have to split it up across several lines.